packages/mui-codemod/src/v6.0.0/sx-prop/test-cases/sx-css-vars.expected.js

```diff
@@ -0,0 +1,69 @@
+<Box
+  sx={theme => ({
+    backgroundImage: "var(--items-imageDark)",
+    ...theme.applyStyles("light", {
+      backgroundImage: "var(--items-imageLight)"
+    })
+  })}
+  style={{
+    "--items-imageLight": items[selectedItemIndex].imageLight,
+    "--items-imageDark": items[selectedItemIndex].imageDark
+  }} />;
+
+<Box
+  style={{
+    ...props.style,
```

Comment:
Shouldn't this come last?

Reply:
No, the generated CSS vars should come last to ensure that it takes effect. It'd be 99.9999% that the CSS vars from `props.style` will get overridden.

Reply:
I don't understand the argument honestly. Regardless of what the `sx` props generated, the users's style attribute should win, even if we almost never have cases where people need to do this.

Reply:
I'm fine to change. As a note, it does not matter which way we choose to override (before or after) because it the generated variable is the same as their existing `style.*`, the result styles will not be the same but as I said, that's 99.9999%.

For example,
**Before**:
```js
<Box
  style={{
    '--items-imageLight': '10px', // existing user's defined variable 
  }}
  sx={{
    fontSize: 'var(--items-imageLight)',
    backgroundImage: (theme) =>
      theme.palette.mode === 'light'
        ? items[selectedItemIndex].imageLight
        : items[selectedItemIndex].imageDark,
  }}
/>;
```

**After**:
```js
<Box
  style={{
    "--items-imageLight": items[selectedItemIndex].imageLight,
    "--items-imageDark": items[selectedItemIndex].imageDark,
    '--items-imageLight': '10px', // existing user's defined variable
  }}
  sx={theme => ({
    fontSize: 'var(--items-imageLight)',
    backgroundImage: "var(--items-imageDark)",
    ...theme.applyStyles("light", {
      backgroundImage: "var(--items-imageLight)", // ❌
    })
  })}
/>;
```